Makefile

```diff
@@ -37,11 +41,12 @@ endif
 ifdef ANDROID_ROOT
 ANDROID := 1
 undefine LINUX
+TCCBRANCH := thirdparty-linux-arm64
```

Comment:
This isn't correct, the `thirdparty-linux-arm64` branch won't work on Android since Android binaries use their own, custom libc. Instead we need to check for CPU arch on non-linuxes (i.e. through `uname`) 

We would need other, Android-libc-specific `thirdparty-android-**` branches, which I haven't gotten to compile yet.

Reply:
That is true, it is one of the problems that this branch still has.